### PR TITLE
Schrul 15499 fix browser windows close without webcontents

### PR DESCRIPTION
**Pull request by phil1618**
* Add defensive checks
* Allow nodeIntegration for Wexond webContents removal close the briwserWindow
* Avoid using setFullscreen in likosk mode

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "flowr-desktop",
-  "version": "1.3.5",
+  "version": "1.3.7",
```
Comment (phil1618) [inline package.json:3]: Version bump

File: src/launcher/index.ts [hunk 2]
```diff
@@ -227,7 +228,10 @@ async function main() {
     try {
       flowrWindow = createFlowrWindow(store, isDebugMode, setDebugMode)
       keyboard.setParentWindow(flowrWindow)
-      FullScreenManager.applyDefaultActionOnWindow(flowrWindow)
+      if (!defaultBrowserWindowOptions(store).kiosk) {
```
Comment (phil1618) [inline src/launcher/index.ts:231]: Do not use setResizable/setFullscreenable is Kiosk mode. This blocks resizing and end up in black margins

File: src/wexond/main/services/web-request.ts [hunk 3]
```diff
@@ -86,8 +86,8 @@ export const loadFilters = async (): Promise<void> => {
 const getTabByWebContentsId = (window: AppWindow, id: number) => {
   for (const key in window.viewManager.views) {
     const view = window.viewManager.views[key]
-
-    if (view.webContents.id === id) {
+   
+    if (view.webContents?.id === id) {
```
Comment (phil1618) [inline src/wexond/main/services/web-request.ts:90]: Defense

File: src/wexond/main/view-manager.ts [hunk 4]
```diff
@@ -185,7 +185,7 @@ export class ViewManager {
 
     // Undocumented electron API: https://github.com/electron/electron/issues/26929#issuecomment-754294324
     // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-    (view.webContents as any).destroy()
+    (view.webContents as any)?.destroy()
```
Comment (phil1618) [inline src/wexond/main/view-manager.ts:188]: Defense

File: src/wexond/main/view.ts [hunk 5]
```diff
@@ -26,6 +26,7 @@ export class View {
         nodeIntegration: false,
         additionalArguments: [`--tab-id=${id}`],
         contextIsolation: true,
+        sandbox: false,
```
Comment (phil1618) [inline src/wexond/main/view.ts:29]: Allow WebContents to close BrowserWindow from within page content

File: src/wexond/renderer/app/store/index.ts [hunk 6]
```diff
@@ -1,157 +1,185 @@
...
       }
-    });
-
-    this.settings = {
-      ...this.settings,
-      ...JSON.parse(readFileSync(getPath('settings.json'), 'utf8')) as Settings,
-    };
+    })
+
+    let localSettingsPath = ''
```
Comment (phil1618) [inline src/wexond/renderer/app/store/index.ts:139]: Defense

**Review by Azevedo-Dani — APPROVED** [review]
LGTM

**Pull request merged**